src/crl.rs

```diff
@@ -46,7 +47,6 @@ use crate::{
 ///   key_identifier_method: KeyIdMethod::Sha256,
 /// };
 /// let crl = CertificateRevocationList::from_params(crl).unwrap();
-/// println!("{}", crl.serialize_pem_with_signer(&issuer).unwrap());
```

Comment:
In this instance it felt better to remove the one line requiring the PEM feature as opposed to using the more cumbersome `cfg` syntax to specify that it's needed.